mobile-app/app/screens/AppNavigator/screens/Settings/screens/AboutScreen.tsx

```diff
@@ -103,6 +105,10 @@ const SOCIAL_LINKS: AboutScreenSocialLinks[] = [
 ]
 
 export function AboutScreen (): JSX.Element {
+  const navigation = useNavigation<NavigationProp<SettingsParamList>>()
+  const { featureFlags } = useFeatureFlagContext()
+  const features = featureFlags.filter((item) => item.stage === 'beta')
```

Comment:
`featureFlags.some()` could be a better option here if you're only interested if an item exist

Reply:
Will update this, Thanks for suggestions